templates/taxi/car_detail.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+{% block content %}
+    <h1>Car information</h1>
+    {% if car.manufacturer.name and car.manufacturer.country%}

```

Comment:
```suggestion
    {% if car.manufacturer.name and car.manufacturer.country %}

```
btw you can't create a manufacturer without name or country